src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs

```diff
@@ -1,12 +1,201 @@
+                case Glyph.ModulePrivate:
+                    return new ImageId(KnownMonikers.ModulePrivate.Guid, KnownMonikers.ModulePrivate.Id);
+                case Glyph.ModuleInternal:
+                    return new ImageId(KnownMonikers.ModuleInternal.Guid, KnownMonikers.ModuleInternal.Id);
+
+                case Glyph.OpenFolder:
+                    return new ImageId(KnownMonikers.OpenFolder.Guid, KnownMonikers.OpenFolder.Id);
+
+                case Glyph.Operator:
+                    return new ImageId(KnownMonikers.Operator.Guid, KnownMonikers.Operator.Id);
+
+                case Glyph.PropertyPublic:
+                    return new ImageId(KnownMonikers.PropertyPublic.Guid, KnownMonikers.PropertyPublic.Id);
+                case Glyph.PropertyProtected:
+                    return new ImageId(KnownMonikers.PropertyProtected.Guid, KnownMonikers.PropertyProtected.Id);
+                case Glyph.PropertyPrivate:
+                    return new ImageId(KnownMonikers.PropertyPrivate.Guid, KnownMonikers.PropertyPrivate.Id);
+                case Glyph.PropertyInternal:
+                    return new ImageId(KnownMonikers.PropertyInternal.Guid, KnownMonikers.PropertyInternal.Id);
+
+                case Glyph.RangeVariable:
+                    return new ImageId(KnownMonikers.FieldPublic.Guid, KnownMonikers.FieldPublic.Id);
+
+                case Glyph.Reference:
+                    return new ImageId(KnownMonikers.Reference.Guid, KnownMonikers.Reference.Id);
+
+                case Glyph.StructurePublic:
+                    return new ImageId(KnownMonikers.ValueTypePublic.Guid, KnownMonikers.ValueTypePublic.Id);
+                case Glyph.StructureProtected:
+                    return new ImageId(KnownMonikers.ValueTypeProtected.Guid, KnownMonikers.ValueTypeProtected.Id);
+                case Glyph.StructurePrivate:
+                    return new ImageId(KnownMonikers.ValueTypePrivate.Guid, KnownMonikers.ValueTypePrivate.Id);
+                case Glyph.StructureInternal:
+                    return new ImageId(KnownMonikers.ValueTypeInternal.Guid, KnownMonikers.ValueTypeInternal.Id);
+
+                case Glyph.TypeParameter:
+                    return new ImageId(KnownMonikers.Type.Guid, KnownMonikers.Type.Id);
+
+                case Glyph.Snippet:
+                    return new ImageId(KnownMonikers.Snippet.Guid, KnownMonikers.Snippet.Id);
+
+                case Glyph.CompletionWarning:
+                    return new ImageId(KnownMonikers.IntellisenseWarning.Guid, KnownMonikers.IntellisenseWarning.Id);
+
+                case Glyph.StatusInformation:
+                    return new ImageId(KnownMonikers.StatusInformation.Guid, KnownMonikers.StatusInformation.Id);
+
+                case Glyph.NuGet:
+                    return new ImageId(KnownMonikers.NuGet.Guid, KnownMonikers.NuGet.Id);
+
+                default:
+                    throw new ArgumentException("glyph");
+            }
+        }
```

Comment:
Nit: The team typically puts a blank line between this close curly brace and the start of the next method.